client/src/web/WebRTC.ts

```diff
@@ -36,32 +73,28 @@ export default class WebRTC {
         // prepare to be called
         this.myPeer.on('call', (call) => {
```

Comment:
is this now duplicated with initialize() above?

Reply:
Yes, the line here (with `myStream` obtained from `getUserMedia()`
```JavaScript
call.answer(this.myStream)
```
overwrites the
```JavaScript
call.answer()
```
in the `initialize()` that allows answering calls even without MediaStream.

Reply:
hmm this feels wrong because you'll have two onCall listener attached.

since you already have myStream saved as a field, can't you just `call.answer(this.myStream)` anyway?

Reply:
ok, that works, I'll fix it! Thanks!